smtp_functions.py

```diff
@@ -0,0 +1,70 @@
+import datetime
+import smtplib
+import typing
+from email.message import EmailMessage
+
+from dynaconf import Dynaconf
+
+
+def create_connection(config: Dynaconf) -> typing.Union[smtplib.SMTP_SSL, smtplib.SMTP]:
+    if "smtp_port" in config.notification:
+        port = config.notification.smtp_port
+    else:
+        port = smtplib.SMTP_PORT
```

Comment:
for SMTP default port is 25, but for SMTP_SSL the default port is 465. So probably better to do `port = 0` here and let smtplib handle the default port choice...